py21cmsense/sensitivity.py

```diff
@@ -94,11 +101,10 @@ def _kconverter(val):
     if not hasattr(val, "unit"):
         # Assume it has the 1/Mpc units (default from 21cmFAST)
         return val * un.littleh / un.Mpc / Planck15.h
-    else:
-        try:
-            return val.to("littleh/Mpc")
-        except un.UnitConversionError:
-            return ((val / Planck15.h) * un.littleh).to("littleh/Mpc")
+    try:
+        return val.to("littleh/Mpc")
+    except un.UnitConversionError:
+        return ((val / Planck15.h) * un.littleh).to("littleh/Mpc")
```

Comment:
I know this part is refactored but it got me thinking, would it be safer to use the `.to("littleh/Mpc", with_H0=Planck15.H0)` equivalency? Since you're already assuming planck15 below it probably isn't much of a stretch to add its equivalency to the conversion. 
https://docs.astropy.org/en/stable/api/astropy.units.with_H0.html#astropy.units.with_H0

Reply:
Oh yeah, that looks nicer.

Reply:
I haven't worked with this before -- does the update look right?

Reply:
I think the change makes the call correctly yeah.